### PR TITLE
data:ShreyanshFaye

### PR DESCRIPTION
**Pull request by Shreyansh-09**

### CODE REVIEW

File: data/Shreyansh-09.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Shreyansh Faye",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Newbie in Web Dev learning frontend with neogcamp",
+  "avatar": "https://avatars.githubusercontent.com/u/68677605?v=4",
```
Comment (Panquesito7) [inline data/Shreyansh-09.json:6]: It's best to use this link as it's shorter and updates automatically.
```suggestion
  "avatar": "https://github.com/Shreyansh-09.png",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data/Shreyansh-09.json (hunk 1 above)
Comment (Shreyansh-09) [reply data/Shreyansh-09.json:6]: yeah sure!

**Review by amandamartin-dev — APPROVED** [review]
LGTM

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Pull request merged**